cmd/genconversion/conversion.go

```diff
@@ -46,7 +46,7 @@ func main() {
 	// TODO(wojtek-t): Change the overwrites to a flag.
 	generator.OverwritePackage(*version, "")
 	for _, knownType := range api.Scheme.KnownTypes(*version) {
-		if !strings.Contains(knownType.PkgPath(), "openshift/origin") {
+		if !strings.Contains(knownType.PkgPath(), "atomic-enterprise/origin") {
```

Comment:
In the deep_copy.go "openshift/origin" is changed to "projectatomic/atomic-enterprise" which seems more correct. Might be good to double check it.